src/Libraries/Microsoft.Extensions.AI.OpenAI/OpenAIModelMapper.ChatCompletion.cs

```diff
@@ -403,7 +403,8 @@ public static ChatCompletionOptions ToOpenAIOptions(ChatOptions? options)
                         jsonFormat.SchemaName ?? "json_schema",
                         BinaryData.FromBytes(
                             JsonSerializer.SerializeToUtf8Bytes(jsonSchema, OpenAIJsonContext.Default.JsonElement)),
-                        jsonFormat.SchemaDescription) :
+                        jsonFormat.SchemaDescription,
+                        jsonSchemaIsStrict: true) :
```

Comment:
Does this require any changes for the inverse `FromOpenAIOptions` mapper so that everything roundtrips properly?